controllers/storagecluster/generate.go

```diff
@@ -2,11 +2,19 @@ package storagecluster
 
 import (
 	"fmt"
+	"math"
+	"strconv"
 
 	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 )
 
+const (
+	StorageDeviceName = "default"
+	MaxPGPerOSD       = 600
```

Comment:
Is 600 the real number? Also why all it a max when it does not donate a maximum value? 
```suggestion
	PGCountPerOSD       = ????
```

Reply:
@ezio-auditore Please do not resolve without an answer

Reply:
Sorry, 600pgs is set in the ceph config
https://github.com/red-hat-storage/ocs-operator/blob/c89d00a2f30fe098fcc1534e2c99096ba95e491c/controllers/storagecluster/cephconfig.go#L35